ct_optcon/include/ct/optcon/solver/lqp/HPIPMInterface.hpp

```diff
@@ -226,20 +226,20 @@ class HPIPMInterface : public LQOCSolver<STATE_DIM, CONTROL_DIM>
 
     //! ocp qp dimensions
     int dim_size_;
-    void* dim_mem_;
+    void* dim_mem_{nullptr};
```

Comment:
Thank you for making those changes.
For consistency, it would make sense to move the inline definition / assigment of nullptr to the *-impl.h file!

Reply:
Done, the nullptr assignments are now in the impl file.